### PR TITLE
[Deff] maint/unused parts modifications

### PR DESCRIPTION
**Pull request by Duny-**
Changelog included.
Next up is making telecomms less vulnerable to engineer stupidity.
ss 2015-02-02 at 10 32 48
In dark green, derelict parts of the station deff was built onto. In lighter green, actual maintenance.

### CODE REVIEW

**Comment by ComicIronic** [comment]
You used map loot spawners?
Also, any important images?

**Comment by Duny-** [comment]
I've been used spawners for a while now. I didn't make any major changes to the layout of the map apart from areas and some doors so there's nothing to screenshot, unless you want to see 200 screenshots of "clean" maintenance.

File: code/game/area/Space Station 13 areas.dm [hunk 1]
```diff
@@ -849,6 +849,48 @@ proc/process_adminbus_teleport_locs()
 	name = "Ghetto Bar"
 	icon_state = "ghettobar"
 
+//Defficiency
+
+/area/maintenance/vault
+	name = "Vault Maintenance"
+	icon_state = "vaultmaint"
+
+/area/derelictparts/fsderelict
```
Comment (IratePirate) [inline code/game/area/Space Station 13 areas.dm]: >derelict

Nigga I will fight you.

Comment (Duny-) [inline code/game/area/Space Station 13 areas.dm]: ss 2015-02-03 at 01 00 27
What better word is there?

**Pull request merged**